src/Umbraco.Web.Common/Extensions/UmbracoBuilderExtensions.cs

```diff
@@ -34,15 +36,20 @@
 using Umbraco.Infrastructure.Runtime;
 using Umbraco.Web.Common.ApplicationModels;
 using Umbraco.Web.Common.AspNetCore;
+using Umbraco.Web.Common.Extensions;
 using Umbraco.Web.Common.Filters;
 using Umbraco.Web.Common.ModelBinders;
 using Umbraco.Web.Common.Profiler;
 using Umbraco.Web.Telemetry;
 using IHostingEnvironment = Umbraco.Core.Hosting.IHostingEnvironment;
 
-namespace Umbraco.Core.DependencyInjection
+namespace Umbraco.Web.Common.Extensions
```

Comment:
`Umbraco.Web.DependencyInjection`?

I would also move it to a folder with the same name.